src/main/java/org/graylog2/alarmcallbacks/jabber/JabberAlarmCallback.java

```diff
@@ -45,22 +46,23 @@
     private static final String CK_REQUIRE_SECURITY = "require_security";
     private static final String CK_RECIPIENT = "recipient";
 
-    private XMPPConnection connection;
+    private AbstractXMPPConnection connection;
```

Comment:
https://github.com/igniterealtime/Smack/wiki/Smack-4.1-Readme-and-Upgrade-Guide
"XMPPConnection is now an interface. Use either AbstractXMPPConnection or one of its subclasses as static type when declaring the connection"